test/dynamo/test_unspec.py

```diff
@@ -10,47 +9,8 @@
 import torch._dynamo.testing
 from torch._dynamo.testing import same
 
-try:
-    from . import test_modules, test_repros
-except ImportError:
-    import test_modules
-    import test_repros
 
-
-def make_unspec_fn(fn):
-    @functools.wraps(fn)
-    def _fn(*args, **kwargs):
-        with torch._dynamo.config.patch("specialize_int_float", False):
-            return fn(*args, **kwargs)
-
-    return _fn
-
-
-def make_unspec_cls(cls):
-    class UnspecTest(cls):
-        pass
-
-    UnspecTest.__name__ = f"Unspec{cls.__name__}"
-
-    for name in dir(cls):
-        if name.startswith("test_"):
-            fn = getattr(cls, name)
-            if not callable(fn):
-                continue
-            new_name = f"{name}_unspec"
-            fn = make_unspec_fn(fn)
-            fn.__name__ = new_name
-            setattr(UnspecTest, name, None)
-            setattr(UnspecTest, new_name, fn)
-
-    return UnspecTest
-
-
-UnspecReproTests = make_unspec_cls(test_repros.ReproTests)
-UnspecNNModuleTests = make_unspec_cls(test_modules.NNModuleTests)
```

Comment:
I've moved the unspec tests over to dynamic shapes since it's simpler to do it that way